connectivity/http/http.c

```diff
@@ -256,14 +251,18 @@ static int ta_http_process_request(ta_http_t *const http, iota_client_service_t
     return process_options_request(out);
   }
 
-  if (api_path_matcher(url, "/mam/[A-Z9]{81}[/]?") == SC_OK) {
-    return process_recv_mam_msg_request(http, iota_service, url, out);
-  } else if (api_path_matcher(url, "/mam[/]?") == SC_OK) {
-    if (payload != NULL) {
-      return process_send_mam_msg_request(http, iota_service, payload, out);
+  if (api_path_matcher(url, "/mam.*") == SC_OK) {
```

Comment:
Alternately, use regex to check recv or sed. Does it make sense?

Reply:
I have no idea about the overhead of `api_path_matcher`. Maybe we should use single query for both recv and send cases.

Reply:
I am afraid this pattern of path being accepted `/mam/trash/messge/send`, so the regex rule of `send` and `recv` are stricter ones but not `.*send[/]?` and `.*recv[/]?`.

Should I change them to `.*send[/]?` and `.*recv[/]?`?

Reply:
sure let me do it!